src/bundle/Resources/public/scss/fieldType/_ezrichtext.scss

```diff
@@ -0,0 +1,55 @@
+@import '../custom';
+
+.ez-field-edit--ezrichtext {
+    .ez-data-source__input {
+        display: none;
```

Comment:
You have to keep in mind, that in won't be focusable when a field is required and has some native validation like: `required` validation.
I found the following styles useful: 
```
opacity: 0;
height: 1px;
```
Thanks to them, you're still able to see native validation errors.